### PR TITLE
fixes most depwarns as of 20180529

### LINKED ISSUE #872: Replace "using" with "import"
per discussion in https://discourse.julialang.org/t/function-name-conflict/10335, it appears that there's a preference for limited `import` of functions as opposed to blanket `using` bringing all function names into the namespace. Here's a good intro issue: determine which functions from each package are required, and `import` them.

### PR DESCRIPTION
**Pull request by sbromberger**
Also closes #872.

### CODE REVIEW

**Comment by sbromberger** [comment]
Passes on -5258, but not on current travis nightly. Will wait and retest.

**Comment by sbromberger** [comment]
Tests are passing locally as well as here. Ready for review and merge.

cc @jpfairbanks @juliohm @slyles1001 @somil55

File: src/linalg/graphmatrices.jl [hunk 1]
```diff
@@ -30,13 +30,13 @@ abstract type Laplacian{T} <: GraphMatrix{T} end
 The standard adjacency matrix.
 """
 struct CombinatorialAdjacency{T,S,V} <: Adjacency{T}
-	A::S
-	D::V
+    	A::S
```
[the remark this answers is not in the document]
Comment (sbromberger) [reply src/linalg/graphmatrices.jl]: It's vscode doing auto-formatting on save. The indentation looks fine to me. Is there something specific you're seeing?

[the remark this answers is not in the document]
Comment (sbromberger) [reply src/linalg/graphmatrices.jl]: Oh, wait. I see the issue. It looks like this file was tabbed originally, but my editor uses 4 spaces. Let me fix.

[the remark this answers is not in the document]
Comment (iamed2) [reply src/linalg/graphmatrices.jl]: It was clearer in the full diff view on Github as it highlighted the spaces as new and the tabs as old

**Comment by juliohm** [comment]
Thanks @sbromberger for the fixes. Is it correct to say that all the changes fit in the 3 categories:

1. Adding package prefixes to function calls like `LinearAlgebra.issymmetric`
2. Replacing `using` with `import`
3. Indentation and code style changes like `<:` becomes ` <: ` with blank space?

I am approving anyways :+1:

**Comment by sbromberger** [comment]
@Juliohm - yes. The formatting changes are a side effect of vscode and autolint. You have the other two explicit goals correct. 

Thanks for the quick review.

**Comment by sbromberger** [comment]
Cc @stefankarpinski since you might be interested in the net effects of this approach.

**Pull request merged**